test/integration/visitor_can_edit_cart_test.rb

```diff
@@ -69,7 +69,7 @@ class VisitorCanEditCartTest < ActionDispatch::IntegrationTest
       within ('.quantity') do
         assert page.has_content?("2")
       end
-      within ('.price') do
+      within ('.subtotal') do
```

Comment:
`(...)` interpreted as grouped expression.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.